assets/css/community.scss

```diff
@@ -6,7 +6,7 @@
         .community-icon { 
```

Comment:
We should probably make a separate css file for affiliates. Right now it's similar enough, but that will probably change and don't want the size of the icons here to affect community page.